### PR TITLE
Replace custom checks from the server script to use concurrently

### PR DESCRIPTION
**Pull request by KshitijThareja**
## Description
This PR replace the use of custom script to handle server startup and termination with concurrently.

#### Issue addressed
#676 

## Changelog

- [#PR no]
  - **Description:** Replaces custom configuration in runVisualTests.js for server startup and termination by concurrently.  Also migrate other essential checks to visual.index.js thereby eliminating the need for a custom script. 
  - **Products impact:** -
  - **Addresses:** #676 
  - **Components:**  -
  - **Breaking:** -
  - **Impacts a11y:** -
  - **Guidance:** -

[#PR no]: PR link

## Steps to test

1. Add the `PERCY_TOKEN` env variable before running tests.
2. Run the tests using `yarn test:visual` to see the new changes in action.

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +66,8 @@
     "npm-run-all": "^4.1.5",
     "nuxt": "2.15.8",
     "prismjs": "^1.27.0",
+    "ps-tree": "^1.2.0",
```
Comment (bjester) [inline package.json]: I suppose you can remove this now

Comment (KshitijThareja) [reply package.json]: Oh yeah. Will make the changes :+1:

**Comment by KshitijThareja** [comment]
Hi @bjester :)
Can this PR be merged now if there are no further changes required?

**Review by bjester — APPROVED** [review]
LGTM

**Pull request merged**